services/wallet/controllers_v3_test.go

```diff
@@ -306,6 +309,12 @@ func TestLinkBitFlyerWalletV3(t *testing.T) {
 		t.Logf("%s, %+v\n", body, err)
 		must(t, "invalid response", fmt.Errorf("expected %d, got %d", http.StatusOK, resp.StatusCode))
 	}
+
+	var res wallet.LinkDepositAccountResponse
+	err = json.NewDecoder(w.Body).Decode(&res)
```

Comment:
Nitpick: It would be better to deal with the response, not the writer itself. Look above, line 306, where we initialise it and use immediately, but it's scoped so is not available here.

```golang
resp := w.Result()
if resp.StatusCode != http.StatusOK { ... }
```

Then here:

```suggestion
	err = json.NewDecoder(resp.Body).Decode(&res)
```


But then we can make one more nitpick and say that data is already in memory, so the use of decoder is not justified (since no streaming occurs), but I am not going to nitpick on that, since it's a test. In production code it would have saved 1 allocation.